llrt_core/src/module_loader/resolver.rs

```diff
@@ -73,7 +73,13 @@ impl Resolver for CustomResolver {
 
         trace!("Try resolve '{}' from '{}'", name, base);
 
-        require_resolve(ctx, name, base, true).map(|name| name.into_owned())
+        require_resolve(ctx, name, base, true).map(|name| {
+            if cfg!(not(windows)) {
+                name.into_owned()
+            } else {
+                name.replace("\\", "/")
+            }
+        })
```

Comment:
We have a "to_abs_path" used by `require_resolve` in various places. Use that method inside of require resolve where this case isn't covered!